src/network/mcpe/protocol/types/event/EntityInteractEventData.php

```diff
@@ -27,6 +27,23 @@
 use pocketmine\network\mcpe\serializer\NetworkBinaryStream;
 
 final class EntityInteractEventData implements EventData{
+	public const TYPE_BREED_ENTITY = 1;
+	public const TYPE_MOUNT_TAMING = 2;
+	public const TYPE_TAME_WOLF_OR_OCELOT = 3;
+	public const TYPE_CREATE_GOLEM = 4; // Place pumpkin block
+	public const TYPE_TRIM_THE_SHEEP = 5;
+	public const TYPE_MILK_A_COW = 6;
+	public const TYPE_SUCCESS_TRADE = 7;
+	public const TYPE_FEED_ENTITY = 8;
+	public const TYPE_SET_FIRE_TO = 9; //TODO: Maybe use FlintAndSteel on Creeper
+
```

Comment:
10 is when putting a dye on an entity.

Reply:
Strange, but I could not find any information about this in the IDA. Hmmmmmm

Reply:
I got that value through websockets, which output events from this packet.